sdcm/utils/k8s/chaos_mesh.py

```diff
@@ -99,16 +107,18 @@ class ExperimentStatus(Enum):
     UNKNOWN = 5
 
 
-class PodChaosExperiment:
-    """Base class for all PodChaos experiments."""
+class ChaosMeshExperiment:
```

Comment:
Is it really going to be suitable for both types of backends? K8S and VMs?

Reply:
The name change here, is that experiments would have a base class,

From where you see it should support VMs ?

Reply:
The name is not specific to K8S, so, one may think it is suitable for any backend.

Reply:
ChaosMesh is specific to chaos-mesh which is available only on K8s.

Reply:
The name is fine for now, 
if we ever use ChaosMesh on VMs, we can revisit the names.